ui/components/ui/contract-token-values/contract-token-values.js

```diff
@@ -47,41 +45,38 @@ export default function ContractTokenValues({
       >
         {tokenName}
       </Typography>
-      <Box className="contract-token-values__copy-address">
-        <Tooltip
-          position="top"
-          title={copied ? t('copiedExclamation') : t('copyToClipboard')}
-        >
-          <ButtonIcon
-            iconName={copied ? ICON_NAMES.COPY_SUCCESS : ICON_NAMES.COPY}
-            className="contract-token-values__copy-address__button"
-            onClick={() => handleCopy(address)}
-          />
-        </Tooltip>
-      </Box>
-      <Box className="contract-token-values__block-explorer">
-        <Tooltip position="top" title={t('openInBlockExplorer')}>
-          <Button
-            type="link"
-            className="contract-token-values__block-explorer__button"
-            onClick={() => {
-              const blockExplorerTokenLink = getAccountLink(
-                address,
-                chainId,
-                {
-                  blockExplorerUrl: rpcPrefs?.blockExplorerUrl ?? null,
-                },
-                null,
-              );
-              global.platform.openTab({
-                url: blockExplorerTokenLink,
-              });
-            }}
-          >
-            <IconBlockExplorer size={24} color="var(--color-icon-muted)" />
-          </Button>
-        </Tooltip>
-      </Box>
+      <Tooltip
+        position="top"
+        title={copied ? t('copiedExclamation') : t('copyToClipboard')}
+      >
+        <ButtonIcon
+          iconName={copied ? ICON_NAMES.COPY_SUCCESS : ICON_NAMES.COPY}
+          color={Color.iconMuted}
+          onClick={() => handleCopy(address)}
+          ariaLabel={copied ? t('copiedExclamation') : t('copyToClipboard')}
+        />
+      </Tooltip>
+      <Tooltip position="top" title={t('openInBlockExplorer')}>
+        <ButtonIcon
+          display={DISPLAY.FLEX}
+          iconName={ICON_NAMES.EXPORT}
+          color={Color.iconMuted}
+          onClick={() => {
+            const blockExplorerTokenLink = getAccountLink(
+              address,
+              chainId,
+              {
+                blockExplorerUrl: rpcPrefs?.blockExplorerUrl ?? null,
+              },
+              null,
+            );
+            global.platform.openTab({
+              url: blockExplorerTokenLink,
+            });
+          }}
+          ariaLabel={t('openInBlockExplorer')}
+        />
+      </Tooltip>
     </Box>
   );
 }
```

Comment:
Removing unneeded CSS and HTML bloat by removing some Box components 

<img width="290" alt="Screenshot 2023-02-08 at 12 40 41 PM" src="https://user-images.githubusercontent.com/8112138/217646228-f76b5986-04a3-4c01-a0a6-5d35cc157282.png">

<img width="703" alt="Screenshot 2023-02-08 at 12 40 12 PM" src="https://user-images.githubusercontent.com/8112138/217646221-714c1520-347c-4c17-a56c-bb2e90d24b4c.png">